fe/fe-core/src/main/java/org/apache/doris/analysis/AggregateInfo.java

```diff
@@ -452,10 +452,14 @@ private void createMergeAggInfo(Analyzer analyzer)  {
         for (int i = 0; i < getAggregateExprs().size(); ++i) {
             FunctionCallExpr inputExpr = getAggregateExprs().get(i);
             Preconditions.checkState(inputExpr.isAggregateFunction());
-            Expr aggExprParam =
-                    new SlotRef(inputDesc.getSlots().get(i + getGroupingExprs().size()));
+            List<Expr> paramExprs = new ArrayList<>();
+            if (inputExpr.fn.functionName().equals("max_by")) {
```

Comment:
It's a tricky workaround, won't merge to production code.

Reply:
Discussed with @HappenLee and @yiguolei , we should change the intermediateType to a list, in another PR, in current PR, we just keep the hard code way.